### ISSUE TITLE
Star rating disappears after selection

### ISSUE THREAD

**Issue by axcho** [woothemes/storefront#278]
When you click on a star rating in the process of submitting a product review, the stars stay empty instead of the appropriate number of stars appearing as filled. So it looks like your click didn't do anything, but if you proceed to click the submit button, it turns out to remember your click correctly. Hovering over the stars works as expected.

I've had several customers report this bug, and it happens when I try it as well. You can see a thread about the issue here: https://wordpress.org/support/topic/problem-with-star-rating-1

**Comment by stuartduff** [comment]
I can replicate this too and confirmed it as an issue in v1.5.3.

**Comment by elvismdev** [comment]
The fix on commit https://github.com/woothemes/storefront/commit/6740e113a06792d8aacd02b7b29d7ed46f13bd23 will be released on Storefront v1.6.0? When that version is coming out?

**Comment by tiagonoronha** [comment]
@elvismdev Yes :)

**Comment by jameskoster** [comment]
@elvismdev Will hopefully be before xmas, if not very early into the new year.

**Comment by superjunky** [comment]
After the last update (1.6.1) , the stars are not visible at all when submitting a review.
I've tested with a fresh Storefront with a clean WP / WC..

Anyone else experiencing this?

**Comment by elvismdev** [comment]
I'm about to take back working on the project where I was using Storefront theme. I'll try to update the theme files to the latest version and will let you know if I experience your same issue.

**Comment by superjunky** [comment]
Thanks. I just did some other testing..
It appears the problem is not in SF, but in my setup. When deactivating all plugins, with a fresh storefront and wc, I don't see the stars.
But when I try it on a fresh WP, it works like expected.

So it's not on your end..

**Comment by elvismdev** [comment]
Great good to know that. Totally it should be something else in your current setup.

**Comment by superjunky** [comment]
What else the languages, plugins or themes could it be?
Can this be database related?

**Comment by BurlesonBrad** [comment]
I ain't the best bug swatter,but.....  @superjunky : 
The stars shine brightly when there's no jquery issues or conflicts. :star2: :star2: :star2: :star2: :star2: 

Now, if 'ya star to see them stars gettin' a bit dim & not working at all, I'd give it a **'right click'** and an **'inspect element'** for any jquery / js conflicts or errors. 

![thats-a-lotta-js](https://cloud.githubusercontent.com/assets/6208757/12963232/fb59c7d6-d00e-11e5-969b-956eb90ec34a.png)
### Whoah Nelly!! That's a lotta .js there fella!!
From David Darmin's HotJar, to AdRoll, Google, Twitter, Facebook, & even New Relic monitoring their demo sites, there's not a single .js error _(you'll see a red circle w/ a white X if there **is** an error)_

@superjunky It probably ain't database related.  More than likely it's a theme or a plugin jackin' up the js funky mojo. The instant the js starts pitchin' a fit, it dominos into breaking **more** things.

Happy :bug: hunting! :+1:

**Comment by superjunky** [comment]
Sorry to bug you again with my star-rating quest..but im totally lost.

1. I installed everthing fresh -> the stars show like they should.
2. I connect my current database to the fresh installation -> the stars disappear..

There are no js conflicts or errors.

I am all out of ideas how to find out what the problem is. I hope anyone can help me, or point me in the right direction.

Thanks!

**Comment by superjunky** [comment]
Btw, when switching to a different theme, the star-rating works. Even though storefront, woocommerce and wordpress are fresh installed, with no plugins.

It MUST be in the database..some setting which only affects storefront.

**Comment by BurlesonBrad** [comment]
@superjunky I ain't the rocket scientist super ninja developer like @stuartduff @mikejolley @jameskoster .  I just come here to stalk these rocket geniuses and hope to God Almighty that I can learn a bit more code as I get dings on my phone, email alerts, and a cute little blue dot at the top of the page.

Now, havin' said all that, you've been forewarned that my heart's a hell-of-a lot bigger than my medulla oblongotta. Whatever skills I got, I've learnt, or I might have are free to share.  

First and foremost, I get that Storefront is free, but 'ya outta be postin' this hooplah over in the forums: https://wordpress.org/support/theme/storefront **Not Here!** I ain't gettin on to 'ya. I've had my ass handed to me about what-to-post where enough times that I figured I'd just tell 'ya the lesson I've had to learn the hard way.

Now, if you'll send me 

1. your site's URL, 
2. a copy of the System Report: WooCommerce > System Status > Get System Report > Copy for Support
3.  and possibly an error log 

to howdy@bradgriffin.me, then I'll try to stop playing with the cache and speed on my site long enough to take a look and see if I can help.  No promises.  Just an offer :+1:
